Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/MicrosoftGraphMail.py

```diff
@@ -1999,6 +2036,27 @@ def send_draft_command(client: MsGraphClient, args):
     return_outputs(f'### Draft with: {draft_id} id was sent successfully.')
 
 
+def update_email_status_command(client: MsGraphClient, args) -> CommandResults:
+    user_id = args['user_id']
+    folder_id = args.get('folder_id')
+    message_ids = argToList(args['message_ids'])
+    status: str = args['status']
+    mark_as_read = (status.lower() == 'read')
+
+    raw_responses = []
+
+    for message_id in message_ids:
+        raw_responses.append(
+            client.update_email_read_status(user_id=user_id, message_id=message_id,
+                                            folder_id=folder_id, read=mark_as_read)
+        )
+
+    return CommandResults(
+        readable_output=f'Emails status has been updated to {status}.',
```

Comment:
```suggestion
        readable_output=f'Emails' status has been updated to {status}.',
```

Reply:
We don't want to say to which emails the status was updated?

Reply:
I originally did, but email IDs look like:
`AAMkAGY3OTQyMzMzLWYxNjltNDE0My05NmZhLWQ5MGY1YjIyNzBkNABGAAAAAACYCKjWAnXBTrnhgWJCcLX7BwDrxRwRjq-zTrN6vWSzK4OWAAAAAAEMAADrxRwRjq-zTrN6vWSzK4OWAAXgdwO0AAA=`

and multiple IDs can be passed to the function, so I thought it'd look overwhelming if IDs are included.

It's possible to print the subject of the email instead, but different emails with different IDs can have the same subject (and when there are replys to an email for example, the same subject is used), so it can be misleading.